### PR TITLE
Advice

### PR DESCRIPTION
**Pull request by codekid92**
First Pull Request, Whoohoo!! Hoping I did this right!

### CODE REVIEW

**Comment by codekid92** [comment]
All conflicts resolved!

File: index.html [hunk 1]
```diff
@@ -1317,10 +1317,13 @@ <h2>Our Advice for Newbies!</h2>
...
       <a href="advice/shanna-walsh-advice.html" target="_blank">Shanna's Advice</a><br>
       <a href="advice/chris-sinclair-advice.html" target="_blank">Chris Sinclair's Advice</a><br>
-      <a href="advice/ash-martindale-advice.html" target="_blank">Ash's Advice</a><br>
+      <a href="advice/ash-martindale-advice.html" target="_blank">Ash's Advice</a><br>advice
+      <a href="advice/alisha-pollitt-advice.html" target="_blank">Alisha Pollitt's advice</a><br>
+
       <a href="advice/kassie-bartow-advice.html" target="_blank">Kassie's Advice 🦋</a><br>
 	    <a href="advice/darryl-j-advice.html">Darryl Jackson’s Advice</a><br>
       <a href="advice/april-jordan-advice.html" target="_blank">April's Advice</a><br>
+main
```
Comment (redrambles) [inline index.html:1326]: Hey @codekid92 ! 

Let's remove this line here - looks like the remnants of conflict markers. ;)

**Comment by redrambles** [comment]
Hey @codekid92 !

Congrats on your first pull request - this is great! I have two little requests for you and then we'll be ready to merge this in.

1) I left a comment on a line of code in the index.html file that needs to be removed - see above.
2) We are missing your `advice/alisha-pollitt-advice.html` file - I'm thinking it just needs to be added, committed and pushed up. If you run `git status`, this should help you confirm. 

Once you have added the advice file and made the change to the line of code, make sure everything is committed and then push up using `git push origin advice` - which will automatically update this pull request. Then we'll be good to go!!

Let us know if you have any questions!

Cheers,
Ann :)

**Comment by twosavoie** [comment]
Hi @codekid92,

Checking in to see if you have any questions about this exercise. Let us know if there's anything we can clarify!

Best,
Lisa

File: index.html (hunk 1 above)
Comment (codekid92) [reply index.html:1325]: Hi Ann, 

Please let me know if I was able to make the correct change. I was not sure if I was supposed to change this in the local advice.html file, and made the changes remotely on GitHub.

**Comment by twosavoie** [comment]
Hi @codekid92,

Great to hear from you! Unfortunately, I don't see that your advice file has been added to the advice folder. You can [see here](https://github.com/skillcrush/git-practice/pull/1166/files) what we have so far. 

When you open the home page in the browser and click on your link, are you taken to your advice page? If not, can you check that you're on the advice branch by running `git branch` in the command line. Then check the advice folder to see if your advice file is inside. 

Once your advice file is inside of the advice folder, rur `git status` to see if there are any changes to be added and committed. You can add the changes with `git add -A` and commit them. Then push the advice branch to GitHub.

If you're still stuck, can you also run send a screenshot of your command line to questions@skillcrush.com. We'll be able to go from there!

Best,
Lisa

**Comment by codekid92** [comment]
Hey Lisa, thanks again for taking a look. 

When I open my homepage in the browser and click on my link, it appears. However when I run git status, I see it is an untracked file which probably explains why it isn't appearing. (See screenshot below) 

Screenshot 2023-11-11 at 5 19 22 PM

I went ahead and added and committed the branch. Would the correct command to push this branch be git- -push- origin- advice.html? I don't want to mess this up any further.

**Comment by twosavoie** [comment]
Hi @codekid92,

Well done! You're right the next step would be to push your code to GitHub. You'll use the command `git push origin advice`. 

When you run the command, you're saying you want to **push** the code on your **advice** branch to the **origin** repo (your GitHub repo.)

I hope that helps! Let us know if there's anything we can clarify! Looking forward to seeing the update!!

Best,
Lisa

**Comment by codekid92** [comment]
Hi Lisa! 

I attempted to push the advice branch and received a "rejected error".  It seems I have changes that I made to  this file on GitHub that I do not have on my command line that I should integrate before pushing? The
Screenshot 2023-11-12 at 8 35 14 PM
 screenshot has all the details!

**Comment by twosavoie** [comment]
Hi @codekid92,

Thanks for the screenshot! It looks like there are changes on your remote repo that aren't on your local repo. I wonder if you made changes to your GitHub repo directly rather than through the command line? If so, a commit would have been created on GitHub that isn't on your local repository.

One of Git's primary jobs is to make sure you don't lose any code - for instance by overwriting it. There are a couple of ways you could approach this.

You could force push your local advice branch (which is fine to do because you're the only person working on this branch right now, so it won't change history for anyone else) by running `git push -f origin advice`

Or you could pull down your remote repo and merge it into your local repo by running `git pull origin main`. Git pull is a combination of git fetch and git merge (you're doing both with a single command) like we did in this lesson: https://learn.skillcrush.com/module-10/finishing-the-forking-workflow/

When you do, there may be some merge conflicts. If so, you'll go into the file in your text editor and choose which version you want to keep. This page of the lesson might help: https://learn.skillcrush.com/module-8/tackling-merge-conflicts/. Once the merge conflicts are resolved, you can add and commit the changes and then push to GitHub. 

How does that sound? Let us know if you have any questions! 

Best,
Lisa

**Comment by codekid92** [comment]
Hello Lisa, 

Would you mind letting me know if my advice file was uploaded? I noticed after adding and committing the changes on my ad
Screenshot 2023-11-14 at 2 11 24 PM
vice branch, the working tree is clean according to the command line. I took a screenshot to show the changes. If not, I will try to go ahead and pull down my remote repo and merge it since I am more familiar with that processes. 

Thanks again for your patience with this long process!

**Comment by twosavoie** [comment]
Hi @codekid92,

Thanks for the screenshot! We don't yet see your advice file in an advice folder. You can see what we see here: https://github.com/skillcrush/git-practice/pull/1166/files.

When you open the git-practice directory in the browser, and click the link to your advice file, do you see your advice page? While still on the advice branch, can you check that your advice file is in the advice folder? Can you send us a screenshot showing your advice file in the advice folder to questions@skillcrush.com? 

If all of that looks good, can you run `git log --oneline`? When you see the commits you've made type `q` to quit and return to your prompt. 

Keep us updated on how you're doing. You've got this!!

Best,
Lisa

**Comment by codekid92** [comment]
Screenshot 2023-11-14 at 7 50 05 PM

My advice file can be found inside the advice folder, yes. If it still isn't appearing, I can go ahead and pull down my remote repo and merge it into my local repo. Maybe giving that a shot will help. 

Thanks

**Comment by twosavoie** [comment]
Hi @codekid92,

This is great! Well done! 

Only one small change to make. You added a css file to the css folder. The next step would be to add a link to that file in the <head> of your advice file. The link will look very similar to the current link to the main.css file but with your advice file's name. (You can keep the link to both files.)

You're super close! Keep going - you're doing great!

Best,
Lisa

**Comment by codekid92** [comment]
Hi Lisa, 

This is something I struggle to understand. Would I be changing the name in the area I have highlighted in the screenshot? Also, would I then have to add and commit the changes? If I understand correctly, this would add changes to my local file which I would have to push over to my remote advice file on GitHub. 


Screenshot 2023-11-15 at 2 11 22 PM

**Comment by twosavoie** [comment]
Hi @codekid92, 

That would be great! You can change the name to your CSS file's name. Or you can add a link to your css file below the link to the main.css file. Either way is completely up to you!

You're right - after making this change you'll add and commit the changes. Then push the advice branch to GitHub which will update your pull request. 

Let us know if you have any questions!

Best,
Lisa

**Comment by codekid92** [comment]
I'm really sorry, but I do not know or understand what I'm supposed to do or what it means. I'm looking back at previous projects and I cannot remember a time where I had to do this and I'm still very unfamiliar with the process. Could I refrain from adding CSS to the file and just keep it as is?

**Comment by twosavoie** [comment]
Hi @codekid92,

Absolutely! You did a great job! And your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

If you do decide to add the link, place this line below the current link to the main.css file: `<link rel="stylesheet" type="text/css" href="../css/alisha-pollitt.css">`. Then issue a pull request with a message that your code is already merged in, and you're updating the link. 

You should be proud of the work you've put into this! Well done!

Best,
Lisa

**Comment by codekid92** [comment]
Thanks Lisa! 

I am putting my HTML and CSS skills into practice and that link is helpful for showing me how to link my css stylesheets.

**Pull request merged**